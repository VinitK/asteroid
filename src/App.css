.w100pc {
    width: 100%;
}

.w80pc {
    width: 80%;
}

.ma {
    margin: auto;
}

.mls {
    margin-left: 10px;
}

.mts {
    margin-top: 10px;
}

.c1 {
    color: #0779e4;
}

.c2 {
    color: #fa744f
}

.c3 {
    color: #16817a;
}

.link {
    text-decoration: none;
}

.button {
    border: none;
    padding: 10px 20px;
    border-radius: 5px;
    color: white;
    background-color: #0779e4;
}

.button:hover {
    filter: brightness(110%);
    cursor: pointer;
}

.frow {
    display: flex;
}

.fcol {
    display: flex;
    flex-direction: column;
}

.fjcsb {
    justify-content: space-between;
}

.faic {
    align-items: center;
}

.fgr1 {
    flex-grow: 1;
}

.textInput {
    padding: 10px;
    border-radius: 5px;
}