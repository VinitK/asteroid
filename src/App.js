import React from 'react';
import {
  BrowserRouter as Router,
  Switch,
  Route
} from "react-router-dom";

import './App.css';
import Topbar from './components/Topbar/Topbar';
import Search from './components/Search/Search';
import Asteroid from './components/Asteroid/Asteroid';

function App() {
  return (
    <Router>
      <div className="App w100pc">
        <Topbar />
        <Switch>
          <Route exact path="/">
            <Search />
          </Route>
          <Route exact path="/:asteroidId">
            <Asteroid />
          </Route>
        </Switch>
      </div>
    </Router>
  );
}

export default App;
