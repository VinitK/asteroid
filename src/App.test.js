import React from 'react';
import { render } from '@testing-library/react';
import { shallow } from 'enzyme';
import App from './App';

describe("App Testing with Jest", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<App />);

  });
  it('App Component Snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
});
