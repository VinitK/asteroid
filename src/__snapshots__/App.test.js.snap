// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Testing with Jest App Component Snapshot 1`] = `ShallowWrapper {}`;
